### ISSUE TITLE
"cancelled-unblocked" message after re-subscribing to block list

### ISSUE THREAD

**Issue by GreatSalamander** [jsha/blocktogether#219]
After a while I decided to re-subscribe to a block list, but after checking the actions tab, a lot of the users say "cancelled-unblocked", instead of "done" or "pending". I assume this is because of my "unblock all" action I took last year. I tried revoking the blocktogether app access, signing in and out, and re-subscribing to the list, but to no avail. Not sure of other actions I could take.

Thanks for your time.

**Comment by jsha** [comment]
You're correct - Block Together will avoid re-blocking users you've previously unblocked. In the case of bulk unblocks this is maybe not the right behavior.

For now, I can clear out the unblock history on your account. First, unsubscribe from any blocks lists, then send a DM to @blocktogether saying "Please clear my unblock history." Once I reply saying it's done, you can re-subscribe and things should work.

**Comment by dtiersch** [comment]
Same issue here. I unblocked all and readded some of my subscriptions. The Actions tab shows "cancelled-unblocked" for most entries now as well with the users staying unblocked :(

**Comment by indecisivemal** [comment]
I made the mistake of importing a blocklist directly to twitter during the downtime, then setting up blocktogether, then unblocking all 900 people hoping blocktogether would take over... Oops. A "clear unblock history" button in the UI would be tremendously helpful, or a "Unblocked users" page for individual removal, but for now I'll DM.
